### DISCUSSION TITLE
How to enable 2160p on plex debrid

### DISCUSSION THREAD

**Discussion by aigars-jekabsons** [itsToggle/plex_debrid#594] [Q&A]
Hello, I just installed plex debrid and plex server on rasperry pi Casaos. Everything works awesomely with a little of tinkering. But I can not figure out how to enable 4k content for docker setup ? Can someone please help out ? :)

**Comment by dancing-skunk** [comment]
You need to enable it via the Scraper Settings -> Versions menu. You can create and customize versions based on a pretty robust rules system
<img width="901" alt="Screenshot 2024-01-06 at 10 26 03 PM" src="https://github.com/itsToggle/plex_debrid/assets/155865120/c75c2f7f-6d0c-49c3-835b-938a61192103">

**Reply by aamorin** [reply]
Would you mind sharing your `2160p HDR` Version configuration @dancing-skunk ? (from the `settings.json` file)
